src/main/java/jenkins/plugins/mattermost/StandardMattermostService.java

```diff
@@ -29,7 +29,7 @@
 	public StandardMattermostService(String endpoint, String roomId, String icon) {
 		super();
 		this.endpoint = endpoint;
-		this.roomIds = roomId.split("[,; ]+");
+		this.roomIds = roomId.split("[,;]+");
```

Comment:
Maybe you misinterpret the regex; what it does is split on `,` and `;`, followed by any number of spaces. It will not split on spaces alone.

Basically, this would be split OK:
* `Bot Name 1@channel1,Bot Name 2@channel2`
* `Bot Name 1@channel1, Bot Name 2@channel2`

This would no longer be split (at all):
* `Bot Name 1@channel1 Bot Name 2@channel2`